### DISCUSSION TITLE
Best POE camera?

### DISCUSSION THREAD

**Discussion by izzygold** [blakeblackshear/frigate#13786] [Ask A Question] [answered]
Hi,
I currently have 8 hikvision non moving POE cameras. the quality of the recording is very good (8MP, good night vision) but the problem is that its practically impossible to configure them without the proprietary recording server it came with. things like focus, toggle nightvision, etc...)

What are your recommendations for great POE cameras? I want great recording/nightvision quality + ability to configure the cameras easily. A "moving" camera is also a possibility for me. 

(And of course I want the camera to work well with frigate)

**Comment by NickM-27** [comment]
are they old hikvision cameras? I don't have any issues with mine using the api to change settings like night mode. The docs have the cameras that are recommended https://docs.frigate.video/frigate/hardware#cameras

**Reply by izzygold** [reply]
any chance to point me to the api docs - specifically changing settings like night mode?

I looked at https://docs.frigate.video/integrations/api but don't see it.

**Reply by izzygold** [reply]
They are hikvision camera's that were whitelabeled and sold as "hwfs". probably older version, maybe custom firmware.

**Reply by hawkeye217** [reply, marked as answer]
https://github.com/loozhengyuan/hikvision-sdk/blob/master/resources/isapi.pdf

**Reply by izzygold** [reply]
wow, nice! thanks!
